errors.js

```diff
@@ -23,7 +23,7 @@ class ERR_INVALID_SHORT_OPTION extends TypeError {
 
 class ERR_UNKNOWN_OPTION extends Error {
   constructor(longOption) {
-    super(`Unknown option: --${longOption}`);
+    super(`Unknown option '${longOption}'`);
```

Comment:
This is where I tried `dashedOption` in another PR. 😄

Reply:
Ah yes, I like that! Would you prefer `dashedOption` or a more explicit `longOrShortOption` param?

Reply:
all options are dashed. long options are double-dashed.

Reply:
Prefer `dashedOption` over `longOrShortOption`. However, I'll add another comment that perhaps not needed! (Which is a bit late given I suggest passing it in...)

Reply:
why not just `option`? i'm saying it's not an option unless it has a dash, so `dashedOption` is redundant.

Reply:
This ended up only being referenced in one place so I inlined the conditional in the error: https://github.com/pkgjs/parseargs/blob/a4fc92af7cea085a016af01e812b812af87c7db9/index.js#L93